### ISSUE TITLE
Dimension compat addition.

### ISSUE THREAD

**Issue by Carvercarver1** [Rebirth-of-the-Night/Ice_and_Fire#9]
You can make it so the structures spawn only in certain biomes like you did with pixies. Roosts and Hydra dens are still technically structures, so it should be possible. Because as of default I&F, Hydras spawn in thebetweenlands and dragons spawn in Atum. It would be a neat addition if it was possible to restrict them via such biome as well as by default any spawning there used a biome stat or value, however since this value is also present in the other modded dimension biomes (like hydra iirc accounts for "swamps", and thebetweenlands is a giant swamp. You can see the point here - needs specific biome restriction because for some reason their default blacklist can't prevent them from spawning there).

**Comment by EliteMeats** [comment]
you can already make dragons only spawn in a certain dimension, but i do agree that the other stuff would be nice

**Comment by SonicX8000** [comment]
Agreed. I dunno what version of Ice & Fire that I had at the time but I did have one of those Myrmexe Hives spawn in the nether.
